runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -114,6 +136,19 @@ initImpl(J9VMThread *currentThread, j9object_t membernameObject, j9object_t refO
 
 		J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(methodID->method);
 
+		J9Class *declaringClass = J9_CLASS_FROM_METHOD(methodID->method);
+		J9UTF8 *methodName = J9ROMMETHOD_NAME(romMethod);
+		if (isPolymorphicMHMethod(vm, declaringClass, methodName)
+#if JAVA_SPEC_VERSION >= 9
+		|| ((declaringClass == J9VMJAVALANGINVOKEVARHANDLE(vm))
+			&& VM_VMHelpers::isPolymorphicVarHandleMethod(J9UTF8_DATA(methodName), J9UTF8_LENGTH(methodName))
+			)
```

Comment:
@DanHeidinga unsure about the location of this bracket. should it be placed on a new line or on line 144?
```suggestion
			&& VM_VMHelpers::isPolymorphicVarHandleMethod(J9UTF8_DATA(methodName), J9UTF8_LENGTH(methodName)))
```

Reply:
Neither are great.  I'd probably put it on the same line (144) but not overly concerned about which we do

Reply:
I've launched builds on this now so if you do decide to change it, wait for the build results first :)